PennyPincherExample/PennyPincherExample/ViewController.swift

```diff
@@ -52,8 +49,61 @@ class ViewController: UIViewController, UITextFieldDelegate {
         recognizerResultLabel.text = "Template: \(template.id), Similarity: \(similarityString)"
     }
    
+    //MARK:- Actions
+    @IBAction func didTapAddTemplate(_ sender: AnyObject) {
+        if
+            let text = templateTextField.text,
+            let template = PennyPincher.createTemplate(text, points: gestureView.points) {
```

Comment:
I would not `guard` in this case, since the lines after this scope need to be run regardless of whether the conditional succeeds, and `guard` will force you to quit its outer scope. If this is preexisting code then there's no harm in leaving it be.